dd-sdk-android/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -34,7 +34,7 @@ import com.datadog.android.telemetry.model.TelemetryConfigurationEvent.ViewTrack
 internal class TelemetryEventHandler(
     internal val sdkCore: SdkCore,
     internal val eventSampler: Sampler,
-    internal val configurationExtraSampler: Sampler = RateBasedSampler(20f.percent()),
+    internal var configurationExtraSampler: Sampler = RateBasedSampler(20f.percent()),
```

Comment:
it doesn't need to be `var` it seems?

Reply:
Yes, thanks. I changed it when I thought I would take a different approach but found one that worked better.